src/components/Popover/set-activator-attributes.ts

```diff
@@ -4,14 +4,20 @@ export function setActivatorAttributes(
   activator: HTMLElement,
   {
     id,
-    active,
+    active = false,
```

Comment:
@dpersing Added the default value for the `active` prop prevents `aria-expanded` being defined as `undefined` on line 18/24.